README.md

```diff
@@ -153,6 +153,12 @@ REST API
 
 ## Repair Run Resource
 
+* GET     /
```

Comment:
This should be the root or RepairRun resource, not the root of the whole host path.